app/components/app_outcome_banner_component.rb

```diff
@@ -48,8 +48,8 @@ def rows
   end
 
   def show_location?
-    # location only makes sense if an attempt to vaccinate on site was made
-    @patient_session.vaccination_records.any?
+    # showing the location only makes sense if an attempt to vaccinate on site was made
+    @patient_session.vaccination_records.recorded.any?
```

Comment:
A part of me wonders if we shouldn't continue with adding default scopes ... maybe something to think about.

Reply:
I think you might be right... I'll try a default scope before merging this PR.

Reply:
https://github.com/nhsuk/manage-childrens-vaccinations/pull/1194

Reply:
Oops I meant https://github.com/nhsuk/manage-childrens-vaccinations/pull/1197